cpu: Remove SLC bit restraint for GPU tester

DanKouch opened the pull request:
This reverts gem5#133, the temporary work-around for gem5#131, allowing both SLC and GLC atomic requests to be made in the GPU tester.

The underlying issues behind gem5#131 have been resolved by gem5#367 and gem5#397.

Change-Id: Iab8817192a4aaae6a691e8ac442acaf1844f1aec

DanKouch commented:
@mattsinc @abmerop The issues which were causing the ruby random tester to fail have been resolved, so #133 can be reverted. I've run the random tester for both write-back and write-through L2s.

mattsinc reviewed (approved):
I'm not sure CPU is the proper tag for this, but it is what @abmerop used in #133, so I guess leave it be to be consistent.

abmerop reviewed (approved):
The tag is mainly due to it being in the cpu directory. I think eventually only the github tags will be used but that hasn't been discussed yet

abmerop commented:
Looks like you need a change ID... If you did not make any local git add's you can do `git commit --amend`, close editor, and it should add one

Commit:
cpu: Remove SLC bit restraint for GPU tester

This reverts gem5#133, the temporary work-around for gem5#131, allowing both
SLC and GLC atomic requests to be made in the GPU tester.

The underlying issues behind gem5#131 have been resolved by gem5#367 and
gem5#397.

Change-Id: Iab8817192a4aaae6a691e8ac442acaf1844f1aec

DanKouch commented:
@abmerop

Added, thanks!.

It looks like the `pre-commit install` command on the contributing page doesn't install hooks for commit-msg (which is what the Change-ID hook uses).

mattsinc commented:
> @abmerop
> 
> Added, thanks!.
> 
> It looks like the `pre-commit install` command on the contributing page doesn't install hooks for commit-msg (which is what the Change-ID hook uses).

Yeah, unfortunately one has to install it separately.

mattsinc commented:
> This reverts #133, the temporary work-around for #131, allowing both SLC and GLC atomic requests to be made in the GPU tester.
> 
> The underlying issues behind #131 have been resolved by #367 and #397.

@DanKouch you will also need to update this "message" box to include the commit ID as I believe this is what shows up in the git commit log.

mattsinc commented:
Seems like the tests were run before adding the ChangeID?  If so, updating the branch to head of develop may kick them off again?

mattsinc commented:
> Seems like the tests were run before adding the ChangeID? If so, updating the branch to head of develop may kick them off again?

Nevermind, with the updates I see now the ChangeID failure is gone (but @DanKouch will need to update to head of develop eventually).
